src/asynchronous.rs

```diff
@@ -222,7 +222,7 @@ impl AsyncDeviceStateReader {
             StreamDeckInput::ButtonStateChange(buttons) => {
                 for (index, (their, mine)) in zip(buttons.iter(), my_states.buttons.iter()).enumerate() {
                     match self.device.kind {
-                        Kind::Akp153 | Kind::Akp153E | Kind::Akp815 | Kind::MiraBoxHSV293S => {
+                        Kind::Akp153 | Kind::Akp153E | Kind::Akp153R | Kind::Akp815 | Kind::Akp03R | Kind::MiraBoxHSV293S => {
```

Comment:
I've also added AKP153R here, because it already had 153 and 153E, and it's pretty much the same device

Reply:
My AKP153R just arrived, I've tested async examples and yep, there is an issue with "Button up" sending only after next button is pressed. But there is also couple more issues with AKP153R, so I would open separate pull request for that one